### DISCUSSION TITLE
How to fetch the "To be registered" documents and  Expense documents using php SDK?

### DISCUSSION THREAD

**Discussion by oraclecode318** [fattureincloud/api#484] [Q&A] [answered]
Hi, everyone.
I hope you are doing well.
I am working on Laravel project by implementing php SDK .
But I can't find scope for fetching "To be registered" documents.
And I also have a problem with fetching Expenses documents.
![image](https://github.com/user-attachments/assets/007c9ff0-5491-471b-a5f6-5c98d9b486e5)

I tried all API examples (ony Get method) but I couldn't find.
Please help me with this problem.

**Comment by valmoz** [comment, marked as answer]
Hello @quantumcoder482 
currently, there isn't a public API for that.

Thanks
Mauro
